src/index/coinstatsindex.cpp

```diff
@@ -0,0 +1,309 @@
+// Copyright (c) 2020 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <chainparams.h>
+#include <coins.h>
+#include <index/coinstatsindex.h>
+#include <serialize.h>
+#include <txdb.h>
+#include <undo.h>
+#include <validation.h>
+
+constexpr char DB_BLOCK_HASH = 's';
+constexpr char DB_BLOCK_HEIGHT = 't';
+
+namespace {
+
+struct DBVal {
+    uint64_t transaction_output_count;
+    uint64_t bogo_size;
```

Comment:
A comment explaining "bogo" would be helpful, I looked at the function that calculates it and it didn't make intuitive sense to me.

Reply:
I think bogo just stands for bogus, indicating that this metric is meaningful in any way other than comparing it to itself, i.e. has the UTXO set grown or shrunk over the last x blocks. I added a comment on the calculation and improved the RPC help with the same text.